llama_stack/apis/post_training/post_training.py

```diff
@@ -177,53 +178,52 @@ class PostTrainingJobArtifactsResponse(BaseModel):
 
 class PostTraining(Protocol):
     @webmethod(route="/post-training/supervised-fine-tune")
-    def supervised_fine_tune(
+    async def supervised_fine_tune(
```

Comment:
can we kill the `PostTrainingSFTRequest` BaseModel? the fields from there are inlined here